src/topology/instances/ennreal.lean

```diff
@@ -498,6 +497,56 @@ by simp only [limsup_eq_infi_supr, inv_map_infi, inv_map_supr, liminf_eq_supr_in
 
 instance : has_continuous_inv ℝ≥0∞ := ⟨order_iso.inv_ennreal.continuous⟩
 
+lemma exists_frequently_lt_of_liminf_ne_top
```

Comment:
This is a strange place for these lemmas, in the middle of the part of the file dedicated to inv. Can you move it to a more sensible location? (Probably at the end of a section or in a new section, possibly at the end of the file if you don't see anything obviously related?)